### ISSUE TITLE
Generate multi-view images from other public datasets without condition

### ISSUE THREAD

**Issue by saltedfisssh** [NVlabs/eg3d#92]
Thank you for releasing the code! When I set cond=False, the model cannot be trained on my custom dataset. And I also get some errors when I try to use the datasets you provided. I observe that all tensors associated with c are 0 dimensional. Are there any other parameters I should set? 
![image](https://user-images.githubusercontent.com/55906118/227780568-f79b9980-bd49-4207-971f-dff7cd6e9fcd.png)

**Comment by Aria-Zhangjl** [comment]
@2777018353 Hi, I got the same problem. Do you know how to deal with it?

**Comment by jasuriy** [comment]
@Aria-Zhangjl  @saltedfisssh did you guys heard from the author or fixed the problem ?

thanks

**Comment by KhoiDOO** [comment]
Same problem too

**Comment by agr78** [comment]
Even if you do not have a conditional dataset, my understanding is that you still need to prepare the `dataset.json` file with the camera parameters vector in a `labels` structure [as outlined here](https://github.com/NVlabs/eg3d/blob/main/docs/training_guide.md#preparing-your-data). I did this and then used `dataset_tool.py` to create my training directory from the images. 

I noticed you still need to set the conditional flag as `cond=True` even if you don't have multiple classes in your training, looks this way from `triplane.py`. 

Also, unrelated, but I believe there is an assumption somewhere in the architecture that the input images have 3 channels. I used `PIL` to convert to RGB. Hope this helps.

**Comment by ChangYuance** [comment]
> Even if you do not have a conditional dataset, my understanding is that you still need to prepare the `dataset.json` file with the camera parameters vector in a `labels` structure [as outlined here](https://github.com/NVlabs/eg3d/blob/main/docs/training_guide.md#preparing-your-data). I did this and then used `dataset_tool.py` to create my training directory from the images.
> 
> I noticed you still need to set the conditional flag as `cond=True` even if you don't have multiple classes in your training, looks this way from `triplane.py`.
> 
> Also, unrelated, but I believe there is an assumption somewhere in the architecture that the input images have 3 channels. I used `PIL` to convert to RGB. Hope this helps.

Yes, I understand the `dataset.json` file is needed. But the question is how to get these parameters about camera poses? Could you please show us your way doing for your own dataset?
